frontend/src/static/js/components/webstatus-line-chart-panel.ts

```diff
@@ -285,7 +285,7 @@ export abstract class WebstatusLineChartPanel extends LitElement {
         .datapoint-details-panel {
           flex-direction: column;
           justify-content: center;
-          align-items: center;
+          align-items: left;
```

Comment:
revert this as the loading state for the task uses this.
```suggestion
          align-items: center;
```

Instead in the missing one implementation chart class do something like this:
```
  static get styles() {
    return [
      super.styles,
      css`
        #missing-one-implementation-datapoint-details-complete {
          display: block;
        }
        .missing-features-table {
          width: 100%;
          overflow-x: auto;
          white-space: nowrap;
        }
      `,
    ];
  }
```